datarepo-clienttests/src/main/java/collector/MeasurementCollector.java

```diff
@@ -0,0 +1,204 @@
+    TimeZone originalDefaultTimeZone = TimeZone.getDefault();
+    TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
+    long startTimeMS = Timestamp.valueOf(startTimestamp).getTime();
+    long endTimeMS = Timestamp.valueOf(endTimestamp).getTime();
+    TimeZone.setDefault(originalDefaultTimeZone);
+
+    collectMeasurements(measurementListFileName, outputDirName, server, startTimeMS, endTimeMS);
+  }
+
+  public static void collectMeasurements(
+      String measurementListFileName,
+      String outputDirName,
+      ServerSpecification server,
+      long startTime,
+      long endTime)
+      throws Exception {
+    // read in measurement list and validate it
+    MeasurementList measurementList = MeasurementList.fromJSONFile(measurementListFileName);
+    measurementList.validate();
+
+    // get an instance of a collector and tell it to execute the measurement list
+    MeasurementCollector collector =
+        new MeasurementCollector(measurementList, server, startTime, endTime);
+    Exception collectorEx = null;
+    try {
+      collector.executeMeasurementList();
+    } catch (Exception ex) {
+      collectorEx = ex; // save exception to display after printing the results
+    }
+
+    collector.writeOutResults(outputDirName);
+
+    if (collectorEx != null) {
+      logger.error("Measurement Collector threw an exception", collectorEx);
+    }
+  }
+
+  public static void printHelp() throws IOException {
+    System.out.println(
+        "Usage (1): ./gradlew collectMeasurements --args=\"measurementListFileName outputDirectoryName\"");
+    System.out.println("  measurementListFileName = file name of the measurement list JSON file");
+    System.out.println(
+        "  outputDirectoryName = name of the same directory that contains the Test Runner results");
+    System.out.println();
+    System.out.println(
+        "Usage (2): ./gradlew collectMeasurements --args=\"measurementListFileName outputDirectoryName serverFileName startTimestamp endTimestamp\"");
+    System.out.println("  measurementListFileName = file name of the measurement list JSON file");
+    System.out.println(
+        "  outputDirectoryName = name of the directory where the results will be written");
+    System.out.println("  serverFileName = name of the server JSON file");
+    System.out.println(
+        "  startTimestamp = timestamp for the start of the interval; format must be yyyy-mm-dd hh:mm:ss[.fffffffff] (UTC timezone)");
+    System.out.println(
+        "  endTimestamp = timestamp for the start of the interval; format must be yyyy-mm-dd hh:mm:ss[.fffffffff] (UTC timezone)");
+    System.out.println();
+    System.out.println(
+        "  e.g. ./gradlew collectMeasurements --args=\"BasicKubernetes.json /tmp/TestRunnerResults\"");
+    System.out.println(
+        "  e.g. ./gradlew collectMeasurements --args=\"BasicKubernetes.json /tmp/TestRunnerResults mmdev.json '2020-08-20 13:18:34' '2020-08-20 13:18:35.615628881'\"");
```

Comment:
I think it would be helpful to add these commands to a readme. 

Also, what is the expected to workflow to get the start/end timestamps?

Reply:
Good point about updating the readme. Also I just realized I never set the description for this PR either. Will do both.

To fetch measurements for a test run, you don't need to get the start/end timestamps explicitly because I included a convenience method that extracts it for you. I'll include the possible workflows in the readme and command help.